shibboleth.php

```diff
@@ -115,6 +123,18 @@ function shibboleth_getenv( $var ) {
 		$var_method . $var_under_upper => TRUE,
 	);
 
+	// If fallback is enabled, we will add the standard environment variables to the end of the array to allow for fallback
+	if ( $fallback ) {
+		$fallback_check_vars = array(
+			$var => TRUE,
+			$var_under => TRUE,
+			$var_upper => TRUE,
+			$var_under_upper => TRUE,
+		);
+
+		array_merge( $check_vars, $fallback_check_vars);
```

Comment:
Need to save the result.
```suggestion
		$check_vars = array_merge( $check_vars, $fallback_check_vars );
```
Alternatively, because it's using keys, you can use: `$check_vars += $fallback_check_vars;`

Reply:
Thanks for catching this @jrchamp. I knew that I missed something 🤦‍♂️